### ISSUE TITLE
Add apple-mobile-web-app-capable meta tag

### ISSUE THREAD

**Issue by a0js** [rmcauley/rainwave#140]
Love the work you've done on this.  Is it possible to add 
`<meta name="apple-mobile-web-app-capable" content="yes">` to the head of the site?  This would enable adding the site to the home screen on iphone as a standalone app.  It would still run on safari, but without the address bar, giving it more of an app like experience.  I'm happy to fork and submit a pull request if you would prefer that.

**Comment by rmcauley** [comment]
Handled graciously by yourself in #141 - thanks a ton. :)

**Comment by williamjacksn** [comment]
I believe this is happening:

https://stackoverflow.com/q/2898740/403554

**Comment by rmcauley** [comment]
Oh wow, that is awful behaviour and an even _more_ awful solution.  Ugh!

**Comment by williamjacksn** [comment]
I agree. And I don’t think you should change all your normal links to JavaScript location changes just to account for this.

**Comment by a0js** [comment]
Yeah I started looking deeper into this...I'm so sorry. I didn't realize it would be this bad. What's more, the standalone web app tosses cookies when it's closed, making them not persistent.  Persistent data is only available using html5 local storage. There is no documentation on the apple developer site to indicate this. I thought it would simply make it run in fullscreen, forgive my naïveté, and please rollback.

**Comment by williamjacksn** [comment]
Don't feel too bad, @a0js. After doing something similar to this for Android, I would have expected a similar experience on iOS. This is so weird.

**Comment by rmcauley** [comment]
Yeah I can't blame you for stumbling into something undocumented!

Reverted in #142.  :(
